### ISSUE TITLE
rate from callerid to rate

### ISSUE THREAD

**Issue by John91000** [iNextrix/ASTPP#491]
Hi

Is it possible to set certain rates?
example
calls to +180001212012 the 0.5 $
but if calls from prefix +304878012 to +1800012485  0.8 $

Possible??

**Comment by John91000** [comment]
can somone answer us?

**Comment by smrdoshi** [comment]
Nope. ASTPP does not have that feature. It will calculate based on
destination number only.

Thanks,

Samir Doshi

iNextrix Technologies Pvt. Ltd.

https://www.inextrix.com

https://www.astppbilling.org

Disclaimer:

The information contained in this communication is confidential and may be
legally privileged. It is intended solely for the use of the individual or
entity to whom it is addressed and others authorized to receive it. If you
are not the intended recipient you are hereby notified that any disclosure,
copying, distribution or taking action in reliance on the contents of this
information is strictly prohibited and may be unlawful. Please notify the
sender immediately and destroy all copies of this message and any
attachments contained in it.

[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
03/07/19,
4:16:42 PM
